examples/cassandra/README.md

```diff
@@ -0,0 +1,320 @@
+
+### Simple Single Pod Cassandra Node
+In Kubernetes, the atomic unit of an application is a [_Pod_](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/pods.md).  A Pod is one or more containers that _must_ be scheduled onto the same host.  All containers in a pod share a network namespace, and may optionally share mounted volumes.  In this simple case, we define a single container running Cassandra for our pod:
+
+```yaml
+id: cassandra
+kind: Pod
+apiVersion: v1beta1
+desiredState:
+  manifest:
+    version: v1beta1
+    id: cassandra
+    containers:
+      - name: cassandra
+        image: kubernetes/cassandra
+        command:
+          - /run.sh
+        cpu: 1000
+        ports:
+          - name: cql
+            containerPort: 9042
+          - name: thrift
+            containerPort: 9160
+        env:
+          - key: MAX_HEAP_SIZE
+            value: 512M
+          - key: HEAP_NEWSIZE
+            value: 100M
+labels:
+  name: cassandra
+```
+
+There are a few things to note in this description.  First is that we are running the ```kubernetes/cassandra``` image.  This is a standard Cassandra installation on top of Debian.  However it also adds a custom [```SeedProvider```](https://svn.apache.org/repos/asf/cassandra/trunk/src/java/org/apache/cassandra/locator/SeedProvider.java) to Cassandra.  In Cassandra, a ```SeedProvider``` bootstraps the gossip protocol that Cassandra uses to find other nodes.  The ```KubernetesSeedProvider``` discovers the Kubernetes API Server using the built in Kubernetes discovery service, and then uses the Kubernetes API to find new nodes (more on this later)
+
+You may also note that we are setting some Cassandra parameters (```MAX_HEAP_SIZE``` and ```HEAP_NEWSIZE```).  We also tell Kubernetes that the container exposes both the ```CQL``` and ```Thrift``` API ports.  Finally, we tell the cluster manager that we need 1000 milli-cpus (1 core).
+
+Given this configuration, we can create the pod as follows
+
+```sh
+$ kubectl create -f cassandra.yaml
+```
+
+After a few moments, you should be able to see the pod running:
+
+```sh
+$ kubectl get pods cassandra
+POD                 CONTAINER(S)        IMAGE(S)               HOST                                                          LABELS              STATUS
+cassandra           cassandra           kubernetes/cassandra   kubernetes-minion-1/1.2.3.4   name=cassandra      Running
+```
+
+
+### Adding a Cassandra Service
+In Kubernetes a _Service_ describes a set of Pods that perform the same task.  For example, the set of nodes in a Cassandra cluster.  An important use for a Service is to create a load balancer which distributes traffic across members of the set.  But a _Service_ can also be used as a standing query which makes a dynamically changing set of Pods available via the Kubernetes API.  This is the way that we use initially use Services with Cassandra.
```

Comment:
maybe say "Pods (or a single Pod in this case)" to clarify that even when there is no load to balance, you want a Service.